source/common/router/config_impl.cc

```diff
@@ -1439,6 +1445,17 @@ VirtualHostImpl::VirtualHostImpl(
     hedge_policy_ = virtual_host.hedge_policy();
   }
 
+  if (!virtual_host.request_mirror_policies().empty()) {
```

Comment:
No reason to guard this, the for loop does nothing if it's empty

Reply:
good point.